src/formatters/InteriorWhitespaceFormatter.ts

```diff
@@ -83,9 +83,16 @@ export class InteriorWhitespaceFormatter {
                 continue;
             }
             isPastFirstTokenOfLine = true;
-            //force token to be exactly 1 space
             if (token.kind === TokenKind.Whitespace) {
-                token.text = ' ';
+                const isDeclaringComponentVariable = (tokens[0].kind === TokenKind.Identifier) && (tokens[0].text === 'm') && (tokens[1].kind === TokenKind.Dot);
```

Comment:
I'm not sure hardcoding `m` is the right approach. What about other variables? Like
- `alpha. beta = true`
- `alpha. m . beta = false`

Reply:
Also, what about when the variables are not the first thing in the line? Like this:
- `print m . foo`
- `doSomething(doSomethingElse(m . foo))`